### ISSUE TITLE
Diffs contain control characters instead of coloring

### ISSUE THREAD

**Issue by stoltene2** [magit/magit#1122]
Unstaged changes:
[1mdiff --git a/karma.conf.js b/karma.conf.js[m
[1mindex bc0e168..01faecb 100644[m
[1m--- a/karma.conf.js[m
[1m+++ b/karma.conf.js[m
[36m@@ -6,6 +6,7 @@[m [mmodule.exports = function(config) {[m
     // base path, that will be used to resolve files and exclude[m
     basePath: '',[m
 [m
[32m+[m
     // testing framework to use (jasmine/mocha/qunit/...)[m
     frameworks: ['jasmine'],[m
 [m

I'm using magit-20131221 from melpa.

the vc-mode diffs work though.  Could I be missing a package?

**Comment by dmwelch** [comment]
+1

**Comment by tarsius** [comment]
https://github.com/magit/magit/wiki/FAQ#diffs-are-not-colorized-and-contain-control-characters

**Comment by dmwelch** [comment]
My bad! This behavior occurred when I tried checking out a branch.  I hadn't messed with bash control chars in forever and completely forgot what they looked like - I originally thought there was some unicode/ascii ugliness going on... 

FWIW, I don't think I would have put the FAQ and my issue together from the title.  I just started using magit, so I went to the Installation page for help before I looked up this issues. :-)
